tomni/cytosmart_data_format/annotations/polygon/main.py

```diff
@@ -168,5 +206,38 @@ def _calculate_roundness(self) -> None:
             self._calculate_area()
 
         _, radius = cv2.minEnclosingCircle(self._contour)
-        enclosing_circle_area = radius ** 2 * np.pi
+        enclosing_circle_area = radius**2 * np.pi
         self._roundness = self._area / enclosing_circle_area
+
+    @staticmethod
+    def __compare_list_points(l1: List[Annotation], l2: List[Annotation]):
+        if len(l1) != len(l2):
+            return False
+
+        start_l2 = 0
+        for i in range(len(l1)):
+            if l1[0] == l2[i]:
+                start_l2 = i
+                break
+
+        sliced_l2 = l2[start_l2:] + l2[:start_l2]
+        print(sliced_l2)
+
+        for i in range(len(l1)):
+            if l1[i] != sliced_l2[i]:
+                return False
+
+        return True
+
+    def __eq__(self, other):
```

Comment:
This function (`__eq__`) is intended for comparing two CDF objects. The utils folder is a good place for this if it's static already. Further, you could be also comparing more types than polygons.

Reply:
i'm blind. lol. The whole time i was assuming this is the main class because you are using lists. So, the whole equality should be just is my polygon == other polygon. There should not be any lists involved here.